CHANGELOG.md

```diff
@@ -7,10 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+- `list_objects`, `download_object_to_path`, `download_object_to_file_object`, `download_folder_to_path`, `upload_from_path`, `upload_from_file_object`, `upload_from_folder` methods in `S3Bucket` - [#85](https://github.com/PrefectHQ/prefect-gcp/pull/175)
+
 ### Changed
 
+- `S3Bucket` additionally inherits from `ObjectStorageBlock` - [#175](https://github.com/PrefectHQ/prefect-gcp/pull/175)
+- Exposed all existing blocks to the top level init - [#175](https://github.com/PrefectHQ/prefect-gcp/pull/175)
+
```

Comment:
Probably should mention aws_client_parameters as a field to AwsCredentials and MinioCredentials